src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -217,32 +217,31 @@ def cli_cosmosdb_collection_delete(client, database_id, collection_id):
 
 def _populate_collection_definition(collection,
                                     partition_key_path=None,
-                                    defaultTtl=None,
+                                    default_ttl=None,
                                     indexing_policy=None):
-    changed = False
+    if not any(not arg is None for arg in [partition_key_path, default_ttl, indexing_policy]):
+        return False
 
-    if partition_key_path:
+    if not partition_key_path is None:
         if 'partitionKey' not in collection:
             collection['partitionKey'] = {}
         collection['partitionKey'] = {'paths': [partition_key_path]}
-        changed = True
 
-    if defaultTtl:
-        changed = True
-        collection['defaultTtl'] = defaultTtl
+    if not default_ttl is None:
```

Comment:
And the same question here. Since the boolean expression changes the truthiness condition here. The empty string is treated as positive condition now. Is it expected?